apps/web/components/sections/project/timeline.tsx

```
@@ -0,0 +1,66 @@
+
+interface TimelineProps {
+  events: TimelineEvent[];
+}
+
+const Timeline: React.FC<TimelineProps> = ({ events }) => {
+  return (
+    <div className="timeline space-y-4 bg-gray-50 rounded-lg p-6 shadow-lg border border-gray-200">
+      <div className="flex items-center justify-between mb-4">
+        <h3 className="text-lg font-semibold text-black">Impact Timeline</h3>
+        <div className="flex space-x-4">
+          <button className="flex items-center text-sm font-medium text-blue-600 hover:underline">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)